### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by francesco-romano**

### CODE REVIEW

**Comment by traversaro** [comment]
@francesco-romano Fixed also on Linux.

**Comment by francesco-romano** [comment]
waiting for travis to merge

**Comment by francesco-romano** [comment]
It should have been fixed now.
If travis passes we can squash everything and merge.

File: .travis.yml [hunk 1]
```diff
@@ -29,17 +29,17 @@ install:
...
-  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo ldconfig; fi
-  - if [ "$TRAVIS_OS_NAME" == "linux" ]; cd ../..; fi
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then git clone https://github.com/robotology/yarp; fi
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then cd yarp; fi
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then mkdir build; fi
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then cd build; fi
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then cmake -DCREATE_LIB_MATH:BOOL=ON -DCREATE_SHARED_LIBRARY:BOOL=ON -DCMAKE_BUILD_TYPE=${TRAVIS_BUILD_TYPE}  ..; fi
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then make; fi
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo make install; fi
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then then sudo ldconfig; fi
```
Comment (traversaro) [inline .travis.yml]: then then

**Comment by francesco-romano** [comment]
This time I launched the complete build on my fork and it was successful. 
Removed all commits and waiting for travis.

**Pull request merged**